packages/docs/docs/docs/testing.md

```diff
@@ -128,3 +131,24 @@ data = encodeCall(
 const timeCrowdsale = await TimeCrowdsale.new()
 await timeCrowdsale.sendTransaction({ data, from: owner })
 ```
+
+## Calling Initialize Functions Manually in Your Unit Tests with Truffle 5
+
+_Note: If you're using Truffle 4, please see 
+[Calling Initialize Functions Manually in Your Unit Tests with Truffle 4](https://docs.zeppelinos.org/docs/advanced.html#calling-initialize-functions-manually-in-your-unit-tests-with-truffle-4) instead._
+
+As of version 5, Truffle uses Web3 version 1, whose Contract object has the ability to
```

Comment:
I wouldn't bother mentioning web3 1.0 here, we just want to tell the user how to call `initialize`